meson_options.txt

```diff
@@ -32,6 +32,7 @@ option('plugin_msr', type : 'boolean', value : true, description : 'enable MSR s
 option('plugin_flashrom', type : 'boolean', value : false, description : 'enable libflashrom support')
 option('plugin_platform_integrity', type : 'boolean', value : false, description : 'enable platform integrity support')
 option('plugin_intel_spi', type : 'boolean', value : false, description : 'enable Intel SPI support')
+option('plugin_tuxec', type : 'boolean', value : false, description : 'enable Tuxedo EC support')
```

Comment:
can you make sure that at least one CI job is running this?

Reply:
I think back a bit; this is waaay too much copy and paste code.

Reply:
@hughsie I agree. That was an experiment to make something work, but it does not look like a correct path. 

We will try with the superio plugin approach as suggested.